drivers/uktty/pl011/Config.uk

```diff
@@ -1,14 +1,12 @@
 menuconfig LIBUKTTY_PL011
 	bool "PrimeCell UART (PL011)"
 	depends on ARCH_ARM_64
+	select LIBUKBOOT
+	select LIBUKOFW
 
 if LIBUKTTY_PL011
+
 menuconfig LIBUKTTY_PL011_EARLY_CONSOLE
```

Comment:
Same here, could become `config` when the base address configuration is removed.